### DISCUSSION TITLE
Mouse support in GITUI

### DISCUSSION THREAD

**Discussion by Node0** [extrawurst/gitui#941] [Ideas]
**Add TUI mouse support (like exists in Midnight Commander)**
It would make browsing through commit history, and other searching operations convenient if the mouse cursor could access them.

**There is no alternative git client implemented as a TUI, let alone any with mouse support**

**Comment by extrawurst** [comment]
mouse support is not really a focus for me, gitui's philosophy is to make keyboard control super easy and fast for everyone to grasp. albeit if someone comes forward and provides a PR I wouldn't block it

**Comment by aTable** [comment]
@Node0  For what it's worth, [lazygit](https://www.youtube.com/watch?v=F7TCWUO9vt4) has mouse support and might be worth a try?

**Comment by pm100** [comment]
been investigating this, for full support we need tui mods. I raised an issue with them but no response.

https://github.com/fdehau/tui-rs/issues/397
